### DISCUSSION TITLE
Not showing preview in LaserGRBL

### DISCUSSION THREAD

**Discussion by JHand080** [arkypita/LaserGRBL#1680] [Q&A] [answered]
Hello. I am new to laser engraving and the first time I hooked my laser (Ortur LM2pro) up to the lasergrbl, it did great. It cut perfect and flawlessly. I hooked it up again last night and now when I click create to view a preview it just gives me 2-3 red lines on the preview. Any idea how I could fix this?

**Comment by StuartB4** [comment]
May need to increase the Quality settings. 
![image](https://user-images.githubusercontent.com/16980026/146654428-3f37f0d7-cad1-4730-a0cd-58c5a979b32b.png)

**Reply by JHand080** [reply]
I have tried this and still nothing!

**Comment by JHand080** [comment]
And when I increase size, I can see it slightly. But it’s like my LaserGRBL is in .10 of a centimeter increments instead of inches now

**Comment by StuartB4** [comment, marked as answer]
LaserGRBL is in mm's. The rulers down the side and along the bottom are mm's, and also in Target image settings.

**Reply by JHand080** [reply]
Ok I had forgotten it was in mm. I was trying to put it as cm. Thank you so so much!!

**Comment by StuartB4** [comment]
Do you have $13 set to 1 in GRBL Configuration.

**Reply by JHand080** [reply]
Yes I do. Still doesn’t work. I appreciate your help. Anything I can do will be appreciated!

**Reply by StuartB4** [reply]
Did you set $13 back to 0 for mm's ?
